Update agenda, reference chats and discussion channels (#4429)
jenkins-infra/jenkins.io: content/events/contributor-summit/index.adoc

The summit will include project updates by the Governance Board, working and special interest groups.
We will also organize breakout sessions to discuss key initiatives in the project and its roadmap.
Topics include but not limited to:

> architecture changes, Jenkins Security, Cloud Native Jenkins, interoperability wint CDF and CNCF projects, documentation, and user experience.

Reviewer: Minor spelling fix
```suggestion
architecture changes, Jenkins Security, Cloud Native Jenkins, interoperability with CDF and CNCF projects, documentation, and user experience.
```

We will have lightning talks and demos of the new and upcoming Jenkins features.
We also organize a track for newcomer contributors who want to participate in the project.

The Jenkins Contributor Summit is hosted by the Continuous Delivery Foundation (CDF) and the Jenkins community.

> int is co-located with link:https://events.linuxfoundation.org/cdcon/[cdCon] on Jun 22-24.

Reviewer: Minor spelling
```suggestion
It is co-located with link:https://events.linuxfoundation.org/cdcon/[cdCon] on Jun 22-24.
```

Newcomer track (link:https://docs.google.com/document/d/1JVbWudREipEF5UJn-bBRU5QIjKf8mzFP9iFdwWbgFB0/edit#heading=h.gio4cq5e5ntv[full agenda])::
Track for newcomers who just want to start contributing to Jenkins or consider doing it.

* Contributing to Jenkins 101
* Introduction to Jenkins plugin development
* Introduction to Jenkins Pipeline and Pipeline Library development
* Jenkins outreach programs and how to participate

End User panel (to be confirmed, 
link:https://docs.google.com/document/d/1JVbWudREipEF5UJn-bBRU5QIjKf8mzFP9iFdwWbgFB0/edit#heading=h.vb3p33as4hwl[full agenda])::
Several key Jenkins users present their case studies, issues they experience and expectations from Jenkins evolution.
These are not demos sessions or feature/company presentations.
Contributors will have an opportunity to ask questions and clarify some points.

Contributor Track (link:https://docs.google.com/document/d/1JVbWudREipEF5UJn-bBRU5QIjKf8mzFP9iFdwWbgFB0/edit#heading=h.ekkv6qocz7b1[full agenda]):: 
The goal of the contributor track is to gather as many people as possible to discuss the next major initiative.
Several topics will be covered simultaneously in breakout rooms.

> We will followe the unconference format.

Reviewer: Spelling fix
```suggestion
We will follow the unconference format.
```

Ignite Talks and Demos (link:https://docs.google.com/document/d/1JVbWudREipEF5UJn-bBRU5QIjKf8mzFP9iFdwWbgFB0/edit#heading=h.yofbvfe396v5[full agenda])::
Series of really short talks and demos, up to 5 minutes each.
The idea is to pitch your project or an idea for future discussion at unconference, anything about Jenkins would work.

We will be accepting demos until the very last minute, make suggestions link:https://docs.google.com/document/d/1JVbWudREipEF5UJn-bBRU5QIjKf8mzFP9iFdwWbgFB0/edit#heading=h.yofbvfe396v5[here]

Happy hour::
Closing event and afterparty.
Details to be announced.

== Discussion channels

=== Chat

We have created the `#jenkins-contributor-summit` channel in the Continuous Delivery Foundation Slack workspace.
Anyone is welcome to join and participate in the organization, planning, and async conversation.
link:/chat/#continuous-delivery-foundation[How to join the CDF Slack].

If you prefer IRC, please use the general purpose `#jenkins` channel on the Libera chat IRC.
link:/chat/#jenkins[More info].

=== Discourse

You can discuss the contributor summit on link:https://community.jenkins.io/[community.jenkins.io].
Please use the `contributor-summit` label.

> You can find all the related topics using link:https://community.jenkins.io/tag/question[this query].

Reviewer: Should this include the contributor-summit tag instead?
```suggestion
You can find all the related topics using link:https://community.jenkins.io/tag/contributor-summit[this query].
```
